src/components/common/ComboBox.vue

```diff
@@ -1,31 +1,72 @@
 <script setup lang="ts">
-import { ErrorObject } from "@vuelidate/core";
+import TextField from "@toruslabs/vue-components/common/TextField.vue";
+import useVuelidate, { ErrorObject } from "@vuelidate/core";
+import { helpers } from "@vuelidate/validators";
 import { computed, ref } from "vue";
 import { useI18n } from "vue-i18n";
 
+import CheckBox from "./CheckBox.vue";
+
 type ItemType = { text: string; value: string };
 
 const props = withDefaults(
   defineProps<{
     label?: string;
     items: ItemType[];
-    modelValue?: string;
+    transferTo?: string;
```

Comment:
This is common component
do not make it a specific component.
Create new component if needed